streamers.json

```diff
@@ -88,5 +92,12 @@
     "speaking": ["EN"],
     "languages": ["Haskell"],
     "schedule": "Streams announced on [Twitter](https://twitter.com/tritlo)"
+  },
+  {
+    "name": "luctielen",
```

Comment:
This isn't ideal, but could you maintain the alphabetical order? (I really should fix this in the README generator)

Reply:
Wasn't aware of the order, fixed it :)